src/index.js

```diff
@@ -3,6 +3,11 @@
 
 /* Node modules */
 import { h, render } from 'preact';
+import 'whatwg-fetch';
+import Promise from 'promise-polyfill';
+if (!window.Promise) {
+		window.Promise = Promise;
+	}
```

Comment:
The colors on the "facebook" elements are not set - so e.g for buttons on the carousel

Reply:
![image](https://user-images.githubusercontent.com/20241539/43000685-cdd4392c-8c22-11e8-9ad1-76d16d4e56ef.png)